Client/mods/deathmatch/logic/CClientPed.cpp

```diff
@@ -6752,6 +6801,57 @@ void CClientPed::ResetVehicleInOut()
 //////////////////////////////////////////////////////////////////
 void CClientPed::UpdateVehicleInOut()
 {
+    if (IsLocalEntity())
+    {
+        // If getting inside vehicle
+        if (m_bIsGettingIntoVehicle)
+        {
+            CClientVehicle* pVehicle = GetRealOccupiedVehicle();
+
+            if (pVehicle)
+            {
+                // Call the onClientVehicleEnter event for the ped
+                // Check if it is cancelled before allowing the ped to enter the vehicle
+                CLuaArguments arguments;
+                Arguments.PushElement(this);                    // player / ped
+                Arguments.PushNumber(m_ucVehicleInOutSeat);     // seat
+
+                if (!pVehicle->CallEvent("onClientVehicleEnter", Arguments, true))
+                {
+                    m_bIsGettingIntoVehicle = false;
+                    RemoveFromVehicle();
+                    return;
+                }
+
+                m_bIsGettingIntoVehicle = false;
+                m_VehicleInOutID = INVALID_ELEMENT_ID;
+                WarpIntoVehicle(pVehicle, m_ucVehicleInOutSeat);
+                SetVehicleInOutState(VEHICLE_INOUT_NONE);
+            }
+        }
+        else if (m_bIsGettingOutOfVehicle)
+        {
+            // If getting out of vehicle
+            CClientVehicle* realVehicle = GetRealOccupiedVehicle();
+            CClientVehicle* networkVehicle = GetOccupiedVehicle();
+
+            if (!realVehicle)
+            {
+                // Call the onClientVehicleExit event for the ped
+                CLuaArguments arguments;
+                Arguments.PushElement(this);                    // player / ped
+                Arguments.PushNumber(m_ucVehicleInOutSeat);     // seat
```

Comment:
change to camelCase, otherwise it will error

Reply:
It will. `Arguments` variable doesnt exist. `arguments` one does.

Reply:
fixed, thanks